stories/index.js

```diff
@@ -106,6 +189,27 @@ storiesOf('LineChart', module)
       </DataProvider>
     ))
   )
+  .add(
+    'min/max (on DataProvider)',
+    withInfo()(() => {
+      const y0Accessor = d => d[1] - 0.5;
+      const y1Accessor = d => d[1] + 0.5;
+      return (
+        <DataProvider
+          defaultLoader={customAccessorLoader}
+          xAccessor={d => d[0]}
+          yAccessor={d => d[1]}
+          baseDomain={staticBaseDomain}
+          series={[
+            { id: 10, color: 'steelblue', y0Accessor, y1Accessor },
```

Comment:
It doesn't seem to pick up the domain calculation here -- both series have the same base domain;

https://www.dropbox.com/s/3j3pf3eble6vb4l/Skjermbilde%202018-06-03%2021.44.15.png?dl=0

Reply:
Wow, well spotted! That would've been a nightmare to debug -- it was a rebase conflict.